<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KApprovalWidget</class>
 <widget class="QWidget" name="KApprovalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1632</width>
    <height>1189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>行政人员审批界面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="kroomreserve.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#KApprovalWidget
{
	background-color: rgb(215, 233, 233);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>预约信息</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="showAllReservation">
            <property name="text">
             <string>查看所有预约</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="logout">
            <property name="text">
             <string>注销登录</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowserAllReserve"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>审批管理</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="showApprovalReserve">
        <property name="text">
         <string>查看待审批预约</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetApprovalReserve"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="agreeReserve">
          <property name="text">
           <string>审批通过</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refuseReserve">
          <property name="text">
           <string>审批否决</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="kroomreserve.qrc"/>
 </resources>
 <connections/>
</ui>
